File: src/my_project/medium_problems/from101to150/unique_paths.py
from typing import List 

def fact(n):  
    return 1 if (n==1 or n==0) else n * fact(n - 1);  

class Solution:
    def uniquePaths(self, m: int, n: int) -> int:
        
        
        numerator = fact(n+m-2)
        denominator = fact(n-1) * fact(m-1)
        return int(numerator/denominator)

File: src/my_project_typescript/medium_problems_typescript/from1to50/uniquePaths.ts
function fact(n:number): number {
    if (n === 0 || n === 1){
        return 1
    }
    else {
        return n*fact(n-1)
    }
}

function uniquePaths(m: number, n: number): number {
    let numerator = fact(n+m-2)
    let denominator = fact(n-1)*fact(m-1)
    return numerator/denominator
};